src/_guides/language/effective-dart/style.md

```diff
@@ -353,9 +368,10 @@ Note that dartfmt does 99% of this for you, but the last 1% is you. It does not
 split long string literals to fit in 80 columns, so you have to do that
 manually.
 
-We make an exception for strings containing URIs&mdash;mainly imports and
-exports. Those can remain single-line strings even if they go over the line
-limit. This makes it easier to search source files for a given path.
+We make an exception for comments strings containing URIs and file
```

Comment:
Are we missing commas or conjunctions here? Also, do we need "mainly imports and exports"? Here's a stab at rewriting it (you can probably do better):

We make exceptions for file paths and for comments and strings that contain URIs.
One advantage of keeping paths on single line is that searching for a particular path is easier.

Reply:
I did leave out a word there. The intent is to permit any combination of:

* comment containing a file path
* comment containing a URI
* string containing a file path
* string containing a URI

Reworded.